README.md

```diff
@@ -61,8 +61,9 @@ Purego uses code that originates from the Go runtime. These files are under the
 License that can be found [in the Go Source](https://github.com/golang/go/blob/master/LICENSE).
 This is a list of the copied files:
 
+* `abi_*.h` from package `runtime/cgo`
 * `zcallback_darwin_*.s` from package `runtime`
-* `internal/abi/abi_*.h` from package `runtime/cgo`
+* `internal/fakecgo/abi_*.h` from package `runtime/cgo`
```

Comment:
So `abi_*.h` and `internal/fakecgo/abi_*.h` are completely the same files, right? If so, can we use symbolic links at the top directory?

Reply:
In other words since the files from other packages are not available a symbolic link fixes nothing

Reply:
Done 👍